src/cloud_manager.ts

```diff
@@ -132,6 +158,23 @@ export default class CloudManager {
         group: InstanceGroup,
         cloudRetryStrategy: CloudRetryStrategy,
     ): Promise<Array<CloudInstance>> {
+        if (group.cloud === 'oracle') {
+            return this.getOracleInstances(ctx, group, cloudRetryStrategy);
+        }
+        if (group.cloud === 'custom') {
+            return this.getCustomInstances();
```

Comment:
I suggest passing the ctx here, to have access to the logger and request ID in case the custom instance handler needs to do anything log-related.

Reply:
I totally agree. I did not do it for linting purposes as I currently don't use the values in the method